plugins/catalog-backend-module-bitbucket-server/api-report.md

```diff
@@ -110,4 +124,50 @@ export type BitbucketServerRepository = {
     }[]
   >;
 };
+
+// @public (undocumented)
+export namespace Events {
```

Comment:
Do we need to expose these to the outside of this plugin?

Reply:
I had it exposed for the use case I had before, as the other plugin I had developed had the same event passed through as bitbucket. Or maybe I thought it needed to be exposed so that the BBSClient could use it.

Reply:
Ah, I am looking through my code now and I just had it exported because the other types listed in the `./lib/types.ts` in this plugin had been exported, so I just followed that same style. Should I not follow this same choice as the other types?

Reply:
I've found that it gives me a warning when I run yarn build:api-docs without having it exported from the plugin index. It won't allow me to build the `api-report.md` if it is not exported.